### PR TITLE
Db 860 unicode fix

### PR DESCRIPTION
**Pull request by jworcestBAH**
We now explicitly use utf-8 through the validation read and write and the job manager's copying of the file.  The problematic D2 file now completes validation (with errors on missing values).

### CODE REVIEW

File: dataactvalidator/filestreaming/csvAbstractReader.py [hunk 1]
```diff
@@ -207,7 +207,6 @@ def _getLine(self):
             self.packetCounter +=1
 
             #Get the current lines
-            packet = packet.decode("utf-8") if not self.isLocal else packet
             currentBytes = self.unprocessed + packet
```
Comment (cmc333333) [inline dataactvalidator/filestreaming/csvAbstractReader.py]: Sanity check: `currentBytes` is supposed to hold a unicode string, not a byte sequence, correct?

File: dataactcore/utils/jobQueue.py [hunk 2]
```diff
@@ -51,16 +52,17 @@ def generate_d_file(api_url, file_name, user_id, d_file_id, interface_holder, ti
...
                 aws_file_name = "".join([str(user_id), "/", timestamped_name])
 
-                with open(timestamped_name, "wb") as file:
+                with open(timestamped_name, "w") as file:
                     # get request
                     response = requests.get(file_url)
                     # write to file
-                    file.write(response.content)
+                    response.encoding = "utf-8"
+                    file.write(str(response.text))
```
Comment (cmc333333) [inline dataactcore/utils/jobQueue.py]: NBD, but `response.text` should be a `str` already, hey?

File: dataactvalidator/filestreaming/csvAbstractReader.py (hunk 1 above)
Comment (jworcestBAH) [inline dataactvalidator/filestreaming/csvAbstractReader.py]: Yes that variable is rather poorly named, it is intended to hold a string, unprocessed is the part of the last packet after the final line break, which is also a string.

File: dataactcore/utils/jobQueue.py (hunk 2 above)
Comment (jworcestBAH) [inline dataactcore/utils/jobQueue.py]: Yes, that's fair, this was originally response.content, which was not a string

**Comment by cmc333333** [comment]
Looks good -- both `CsvS3Reader` and `CsvLocalReader` are reading and writing unicode content.

**Pull request merged**